File: State-of-artPizza/State-of-artPizza/Pizza.cpp
#include "Pizza.h"

void Pizza::bake()
{
    cout << "baking pizza" << endl;
}

void Pizza::cut()
{
    cout << "cutting pizza" << endl;
}

void Pizza::box()
{
    cout << "packing the pizza into box" << endl;
}

string Pizza::getName()
{   
    return name;
}

void Pizza::setName(const string& name)
{
    this->name = name;
}

File: PizzaWorld/src/Pizza/Pizza.java
package Pizza;

import java.util.ArrayList;
import java.util.List;

public abstract class Pizza {

    String name;
    String dough;
    String sauce;
    List<String> toppings = new ArrayList<String>();

    public String getName(){
        return name;
    }

    public void prepare() {
        
    }

    public void bake() {

    }

    public void cut() {

    }

    public void box() {

    }
    
}